### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kmjhej**

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [58efefaaa2e3ce6e3c293ef6cfa58991dfc9dc69] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [58efefaaa2e3ce6e3c293ef6cfa58991dfc9dc69] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [e7a13e787b3024af969a15083cff9ae1acaeb23a] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [e7a13e787b3024af969a15083cff9ae1acaeb23a] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [9204f9b964116ceaa8468fd52158d7685a29e147] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [9204f9b964116ceaa8468fd52158d7685a29e147] - Code Rule Check OK.

**Comment by sunghan-chang** [comment]
@kmjhej Thank you for contribution. But you had not better fix cjson file by our policy.

File: external/json/cJSON.c [hunk 1]
```diff
@@ -416,10 +389,8 @@ static unsigned char* ensure(printbuffer * const p, size_t needed)
     }
     p->length = newsize;
     p->buffer = newbuffer;
-
     return newbuffer + p->offset;
 }
-
```
Comment (shadaksharikp) [inline external/json/cJSON.c]: As per the coding guidelines One line spacing between function is recommended for readability

May i know why it has been removed ?

File: external/json/cJSON.c [hunk 2]
```diff
@@ -429,10 +400,8 @@ static void update_offset(printbuffer * const buffer)
         return;
     }
     buffer_pointer = buffer->buffer + buffer->offset;
-
     buffer->offset += strlen((const char*)buffer_pointer);
 }
-
```
Comment (shadaksharikp) [inline external/json/cJSON.c]: Same as above

File: external/json/cJSON.c [hunk 3]
```diff
@@ -489,22 +452,17 @@ static cJSON_bool print_number(const cJSON * const item, printbuffer * const out
...
         output_pointer[i] = number_buffer[i];
     }
     output_pointer[i] = '\0';
-
     output_buffer->offset += (size_t)length;
-
     return true;
 }
-
```
Comment (shadaksharikp) [inline external/json/cJSON.c]: same as above

File: external/json/cJSON.c [hunk 4]
```diff
@@ -524,17 +482,14 @@ static unsigned parse_hex4(const unsigned char * const input)
...
         if (i < 3)
         {
             /* shift left to make place for the next nibble */
             h = h << 4;
         }
     }
-
     return h;
 }
-
```
Comment (shadaksharikp) [inline external/json/cJSON.c]: Same as above

File: external/json/cJSON.c [hunk 5]
```diff
@@ -648,29 +593,23 @@ static unsigned char utf16_literal_to_utf8(const unsigned char * const input_poi
...
     }
-
     *output_pointer += utf8_length;
-
     return sequence_length;
-
 fail:
     return 0;
 }
-
```
Comment (shadaksharikp) [inline external/json/cJSON.c]: Same as above

**Comment by sunghan-chang** [comment]
@kmjhej This fix is not right. Let's close now, when you are free, please fix again and reopen this. Thank you for contribution.

**Pull request closed without merging**